python/hsfs/feature_group.py

```diff
@@ -2911,9 +2915,6 @@ def as_of(
                 .as_of("2020-10-20", exclude_until="2020-10-19")
             ```
 
-        !!! warning
-            This function only works for feature groups with time_travel_format='HUDI'.
-
```

Comment:
you still need time travel enabled feature group, but we now also support time_travel_format='DELTA'